features/steps/steps.py

```diff
@@ -441,6 +475,45 @@ def when_i_replace_string_in_file_with_token(
     when_i_replace_string_in_file(context, original, filename, token)
 
 
+@when(
+    "I replace `{original}` in `{filename}` with commands to install the `{series}` ua version under test"  # noqa: E501
+)
+def when_i_replace_string_in_file_with_install_commands(
```

Comment:
@lucasmoura This is a good idea! And it is something I looked into briefly but didn't go too far. Maybe you can help though, because my lack of apt knowledge held me back a bit here.

Is there an easy way to `apt download` a `xenial` package when on a `focal` machine?

I'd rather keep the host machine of this test `focal` and just change the container base series version. The main reason is that `xenial`'s `docker` doesn't support mount secrets, which are necessary for this workflow :/

Reply:
I think we can use `ubuntu-dev-tools` for this. The command `pull-lp-debs ubuntu-advantage-tools xenial` should download the latest version of the package (even if it is on proposed). It also seems that we can specify a ppa to pull the package from, so maybe we can adapt it for the daily case. But it is worth testing it out

Reply:
oh very cool! I'll definitely test that out - thanks!